src/reanimated2/Colors.ts

```diff
@@ -7,7 +7,8 @@
  */
 
 /* eslint no-bitwise: 0 */
-
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
 import { Platform } from 'react-native';
```

Comment:
Is this to ignore the missing import? Instead of writing this all over the place, you can just install the types for `react-native` as a devDependency:

```sh
npm i @types/react @types/react-native --save-dev
```

Reply:
No, this is ignoring entire file. This PR mostly changes extension of files, so I won't be dealing with conflicts later on